cob_light/common/include/modeExecutor.h

```diff
@@ -67,26 +68,31 @@ class ModeExecutor
 	ModeExecutor(IColorO* colorO);
 	~ModeExecutor();
 
-	void execute(Mode* mode);
-	void execute(cob_light::LightMode requestMode);
+	uint64_t execute(boost::shared_ptr<Mode> mode);
+	uint64_t execute(cob_light::LightMode requestMode);
 
 	int getExecutingPriority();
 	int getExecutingMode();
+	uint64_t getExecutingUId();
 
+	void pause();
+	void resume();
```

Comment:
correct. The user is only able to start new modes with a respective priority and stop running modes if he tracked them